cluster/saltbase/salt/generate-cert/init.sls

```diff
@@ -10,7 +10,7 @@
     {% set cert_ip='_use_azure_dns_name_' %}
   {% endif %}
   {% if grains.cloud == 'vagrant' %}
-    {% set cert_ip=grains.ip_interfaces.eth1[0] %}
+    {% set cert_ip=grains.api_servers %}
```

Comment:
I am biased to remove this if block altogether.

We are passing in the cert_ip explicitly now here:
https://github.com/GoogleCloudPlatform/kubernetes/blob/master/cluster/vagrant/provision-master.sh#L108

So the if pillar['cert_ip'] is always taking precedence now.